app/component/search/search-two-fields-container.cjsx

```diff
@@ -26,7 +32,10 @@ class SearchTwoFieldsContainer extends React.Component
     @context.getStore('PositionStore').removeChangeListener @onGeolocationChange
 
   onGeolocationChange: =>
-    @forceUpdate()
+    #We want to rerender only if position status changes,
+    #not if position changes
+    if @positionStatusChanged
+      @setState({positionState: @context.getStore('PositionStore').getLocationState()})
```

Comment:
We don't need to keep the state, as we get statusChanged as parameter from the event